File: Form_ToPracticom/Form_ToPracticom/Entity/Parent.cs
using System;
using System.Collections.Generic;

namespace Repositories.Entity;

public partial class Parent
{
    public int Id { get; set; }

    public string? IdNumberParent { get; set; }

    public string? FirstName { get; set; }

    public string? LastName { get; set; }

    public DateTime? DateOfBirth { get; set; }

    public string? MaleOrFemale { get; set; }

    public string? Hmo { get; set; }

    public virtual ICollection<Child> Children { get; } = new List<Child>();
}

File: angular without node-module/src/Models/Parent.ts
import Child from "./Child";

export default class Parent{
    constructor( 
      public id :number, public idNumberParent:string,public firstName:string,
      public lastName:string,public dateOfBirth:Date,public maleOrFemale:string,
      public hmo:string,public children:Child[]
     ){}
}